README.md

```diff
@@ -214,7 +214,8 @@ There is no need to use `docker login` or `docker logout`.
 
 ## Troubleshooting
 
-Logs from the Amazon ECR Docker Credential Helper are stored in `~/.ecr/log`.
+Logs from the Amazon ECR Docker Credential Helper are stored inside you cache
+files directory (e.g.: `~/.cache/ecr/log` on *nix).
```

Comment:
Let's either link to the [Go documentation](https://golang.org/pkg/os/#UserCacheDir) or describe the specific locations for Linux, Windows, and Mac.